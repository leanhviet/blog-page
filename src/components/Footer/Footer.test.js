// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { FooterBase } from './index'

describe('FooterBase component', () => {
  const wrapper = shallow(<FooterBase />)

  // Testing Snapshot
  it('should be render snapshot FooterBase component', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
