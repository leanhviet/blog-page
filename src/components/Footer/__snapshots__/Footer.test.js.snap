// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterBase component should be render snapshot FooterBase component 1`] = `
<footer
  className="page-footer unique-color-dark"
>
  <div
    className="footer-copyright py-3 text-center"
  >
    <p>
      Â© 2020 Copyright: Viet Le
    </p>
  </div>
</footer>
`;
